docs/topics/classes.md

```diff
@@ -101,10 +101,9 @@ Learn more about [visibility modifiers](visibility-modifiers.md#constructors).
 The class can also declare _secondary constructors_, which are prefixed with `constructor`:
 
 ```kotlin
-class Person {
-    var children: MutableList<Person> = mutableListOf()
-    constructor(parent: Person) {
-        parent.children.add(this)
+class Pet {
```

Comment:
Do we suppose that we have got already declared `Person` class with the collection of Pet objects?

If we don't need any explanation of this (or it was somewhere above) it looks good.